### PR TITLE
Add SMTP credentials for dev/staging

### PR DESCRIPTION
**Pull request by albertchae**
## Summary of the problem

Part of https://github.com/hackclub/hcb/issues/4927

Staging login is broken after development credentials were made the default on staging. Instead of restoring prod credentials to staging, we decided to make a separate set of credentials for staging for better separation in case we need to revoke access.

## Describe your changes

This PR adds smtp username, password, address, domain and port to the development credentials file for the IAM user `bank-staging-hackclub-smtp-ses`

One gotcha we encountered was a `535 Authentication Credentials Invalid`, most likely due to https://stackoverflow.com/questions/45653939/amazon-ses-535-authentication-credentials-invalid-trying-to-rotate-access-key/45654695#45654695

### CODE REVIEW

**Comment by albertchae** [comment]
Should we change the email domains in https://github.com/hackclub/hcb/blob/7606060784aaa7dd9eaad15245941194f57c5b9f/app/mailers/login_code_mailer.rb#L9-L10 too @garyhtou ?

**Comment by garyhtou** [comment]
> Should we change the email domains in
> 
> https://github.com/hackclub/hcb/blob/7606060784aaa7dd9eaad15245941194f57c5b9f/app/mailers/login_code_mailer.rb#L9-L10
> 
> too @garyhtou ?

Yes! Should we change both? or just `hcb@hackclub.com`

**Comment by albertchae** [comment]
> > Should we change the email domains in
> > https://github.com/hackclub/hcb/blob/7606060784aaa7dd9eaad15245941194f57c5b9f/app/mailers/login_code_mailer.rb#L9-L10
> > 
> > too @garyhtou ?
> 
> Yes! Should we change both? or just `hcb@hackclub.com`

Hm, honestly not sure. I'll do both for now

File: app/mailers/login_code_mailer.rb [hunk 1]
```diff
@@ -4,10 +4,12 @@ class LoginCodeMailer < ApplicationMailer
...
     @pretty_login_code = pretty_login_code
 
+    domain = Rails.env.production? ? "hackclub.com" : "staging.hcb.hackclub.com"
+
     mail(to: email_address,
          subject: "Hack Club Login Code: #{@pretty_login_code}",
-         from: "Hack Club <login@hackclub.com>",
-         reply_to: "HCB <hcb@hackclub.com>")
+         from: "Hack Club <login@#{domain}>",
+         reply_to: "HCB <hcb@#{domain}>")
```
Comment (albertchae) [inline app/mailers/login_code_mailer.rb]: image

Comment (garyhtou) [reply app/mailers/login_code_mailer.rb]: Yay!

**Review by garyhtou — COMMENTED** [review]
Can we also update https://github.com/hackclub/bank/blob/f3c5debbebe9b7201a0887350b77923db9f2f6f7/app/mailers/application_mailer.rb#L6-L6

File: app/mailers/application_mailer.rb [hunk 2]
```diff
@@ -3,7 +3,8 @@
 class ApplicationMailer < ActionMailer::Base
   OPERATIONS_EMAIL = "hcb@hackclub.com"
```
Comment (albertchae) [inline app/mailers/application_mailer.rb]: didn't change this one per our slack DM @garyhtou

**Review by garyhtou — APPROVED** [review]
Great, thanks!

**Pull request merged**